README.md

```diff
@@ -192,7 +192,7 @@ helm show values hedera-json-rpc-relay
 Deploy an installation with custom values file:
 
 ```
-helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./helm-chart --debug
+helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./charts --debug
```

Comment:
```suggestion
helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./charts/hedera-json-rpc --debug
```

Reply:
I made this mistake but will fix it in main

Reply:
yeah, I will update thos PR to include that.